### DISCUSSION TITLE
How to change scratch addons' url from scratch.mit.edu to something other

### DISCUSSION THREAD

**Discussion by musafiroon** [ScratchAddons/ScratchAddons#3706] [Help]
I have made the scratch GUI work on my localhost.  
So I want to access all features ( for editor ) in my localhost.   
I tried to change the manifest.json
But it is giving me errors.
So, could you tell me that how should I access it from localhost?

**Comment by cobaltt7** [comment]
You would have to do a global find and replace.


________________________________
From: musafiroon ***@***.***>
Sent: Monday, October 25, 2021 2:24:02 AM
To: ScratchAddons/ScratchAddons ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ScratchAddons/ScratchAddons] How to change scratch addons' url from scratch.mit.edu to something other (Discussion #3706)


I have made the scratch GUI work on my localhost.
So I want to access all features ( for editor ) in my localhost.
I tried to change the manifest.json
But it is giving me errors.
So, could you tell me that how should I access it from localhost?
